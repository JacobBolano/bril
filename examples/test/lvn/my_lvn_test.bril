@main() {
  a: int = const 42;
  b: int = const 35;
  c: int = add a b;
  d: int = add a b;
  e: int = add c d;
  print e;
}
