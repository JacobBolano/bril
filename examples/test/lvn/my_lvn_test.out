@main() {
  e: const 154;
  print e;
}
